rust/src/database/verkle/variants/managed/mod.rs

```diff
@@ -10,7 +10,5 @@
 
 mod nodes;
 
-pub use nodes::{
-    Node, NodeType, empty::EmptyNode, id::NodeId, inner::InnerNode, leaf::FullLeafNode,
-    sparse_leaf::SparseLeafNode,
-};
+#[allow(unused)]
+pub use nodes::NodeFileStorageManager;
```

Comment:
I noticed that I cannot jump to this or `TestNodeFileStorageManager` (i.e., the location of the derive macro). This works for `mockall:automock`, is there something that can be done about it? Also the docblock is not shown on hover.